src/app/components/Metadata/index.jsx

```diff
@@ -16,6 +16,7 @@ const Metadata = ({
   articleSection,
   brandName,
   canonicalLink,
+  canonicalLinkGb,
```

Comment:
Surely we need to actually pass this from the container?

Reply:
🤦‍♂️